lib/spack/spack/schema/container.py

```diff
@@ -54,7 +54,7 @@
         "os_packages": {
             "type": "object",
             "properties": {
-                "command": {"type": "string", "enum": ["apt", "yum"]},
+                "command": {"type": "string", "enum": ["apt", "yum", "zypper"]},
```

Comment:
```suggestion
                "command": {"type": "string", "enum": ["apt", "yum", "zypper", "apk", "yum_amazon"]},
```

Reply:
You're right. Let me update my pull request.